rest/v9/guild.ts

```diff
@@ -303,6 +304,11 @@ export type RESTPatchAPIGuildChannelPositionsJSONBody = Array<{
  */
 export type RESTPatchAPIGuildChannelPositionsResult = never;
 
+/**
+ * https://discord.com/developers/docs/resources/guild#list-active-threads
+ */
+export type RESTGetAPIGuildThreadsResult = Omit<APIThreadList, 'has_more'>;
```

Comment:
Can't you just remove the has_more since it's getting yeeted anyways?

Reply:
the get channel (public/private/joined private) archived threads routes are keeping it, it's only removed from get guild active threads